skyportal/plot.py

```diff
@@ -454,7 +454,13 @@ def photometry_plot(obj_id, user, width=600, height=300):
     )
 
     # Mark the first and last detections again
+    data['hasmag'] = ~data['mag'].isna()
+    detection_dates = data[data['hasmag']]['mjd']
     if len(detection_dates) > 0:
+        first = round(detection_dates.min(), 6)
+        last = round(detection_dates.max(), 6)
+        first_x = np.full(5000, first)
+        last_x = np.full(5000, last)
```

Comment:
Are these used anywhere?

Reply:
They were, but only once so I just moved these to be inline